modules/virt-creating-local-block-pv.adoc

```diff
@@ -18,25 +17,24 @@ virtual machine image.
 
 .Procedure
 
-. Log in as `root` to the node on which to create the local PV. This procedure
+. Log in as root user to the node on which to create the local PV. This procedure
 uses `node01` for its examples.
 
 . Create a file and populate it with null characters so that it can be used as a block device.
 The following example creates a file `loop10` with a size of 2Gb (20 100Mb blocks):
 +
 [source,terminal]
 ----
-$ dd if=/dev/zero of=<loop10> bs=100M count=20
+# dd if=/dev/zero of=loop10 bs=100M count=20
 ----
 
-. Mount the `loop10` file as a loop device.
+. Mount the `loop10` file as a loop device:
 +
 [source,terminal]
 ----
-$ losetup </dev/loop10>d3 <loop10> <1> <2>
+# losetup /dev/loop10 loop10 <1>
```

Comment:
Do you mean that the path and the file name are now one parameter where previously it was two?

Reply:
No, they are still 2 parameters, but with 1 callout explaining what they are. Multiple callouts on the same line of code look messy.

Reply:
Also "d3" appears to have been an error that was introduced a while ago and never caught.